addons/account/models/res_currency.py

```diff
@@ -20,19 +20,10 @@ def _get_fiscal_country_codes(self):
     @api.depends('rounding')
     def _compute_display_rounding_warning(self):
         for record in self:
-            record.display_rounding_warning = record.id \
+            record.display_rounding_warning = record._origin.id \
                                               and record._origin.rounding != record.rounding \
                                               and record._origin._has_accounting_entries()
 
-    def write(self, vals):
-        if 'rounding' in vals:
-            rounding_val = vals['rounding']
-            for record in self:
-                if (rounding_val > record.rounding or rounding_val == 0) and record._has_accounting_entries():
-                    raise UserError(_("You cannot reduce the number of decimal places of a currency which has already been used to make accounting entries."))
```

Comment:
Hey @dijo-odoo ,
Thanks for working on this.
Is there any reason to remove the constraint on reducing the decimal accuracy ? It was put in place to avoid the customer reducing decimal accuracy then complaining about things like rounding and reconciliation issues.

This also goes against the following warning message :sweat_smile: 
> "This change cannot be undone without technical support."